jvb/src/main/resources/reference.conf

```diff
@@ -321,8 +321,11 @@ videobridge {
   }
 
   version {
-    // Wheather to announe the jitsi-videobridge version to clients in the ServerHello message.
+    # Wheather to announe the jitsi-videobridge version to clients in the ServerHello message.
```

Comment:
Typos: "Whether", "announce".  This is old code but might as well fix it when you're editing the line?